examples/utils/example-utils/src/modelLoader/modelLoader.ts

```diff
@@ -72,7 +62,10 @@ export class ModelLoader<ModelType> implements IModelLoader<ModelType> {
 	// TODO: Consider making the version param optional, and in that case having a mechanism to query the codeLoader
 	// for the latest/default version to use?
 	public async createDetached(version: string): Promise<IDetachedModel<ModelType>> {
-		const container = await this.loader.createDetachedContainer({ package: version });
+		const container = await createDetachedContainer({
+			codeDetails: { package: version },
+			...this.props,
```

Comment:
Order should be flipped, the explicitly provided code details here should take precedence.